### ISSUE TITLE
Test validation process seems invalid (test_set is using train directory instead of test directory)

### ISSUE THREAD

**Issue by JordanMicahBennett** [JohnChangUK/Pneumonia-Kaggle#1]
Good work on the project. 

However, the "test_set " variable in line [21] [from your pneumonia notebook](https://github.com/JohnChangUK/Pneumonia-Kaggle/blob/master/pneumonia.ipynb), seems to be incorrectly defined.

It looks like the "**test_dataGen.flow_from_directory**" function's first parameter should be set to 'xray_dataset/test', instead of 'xray_dataset/train'. 

The test set looks like it was accidentally set by original author as the training set, and in the end this would not reflect actual accuracy of model.

Snippet for quick view on issue:

![image](https://user-images.githubusercontent.com/3666405/76042595-37a0ca80-5f23-11ea-98cb-de2c40c6569d.png)

**Comment by JohnChangUK** [comment]
Thanks for pointing this out Jordan, you're correct that it should be using the test data =)!

**Comment by JordanMicahBennett** [comment]
> Thanks for pointing this out Jordan, you're correct that it should be using the test data =)!

Of course :)
